### ISSUE TITLE
Color contrast and high contrast issue observed for the dropdown arrow

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4642]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 1.4.3 Contrast (Minimum)

**Test Environment:**
- Test URL: https://clarity.design/documentation/forms
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0

**Steps to reproduce:**
1. Open the test URL.
2. Check the color contrast of the dropdown arrow, which is informative and tells sighted users that it's a dropdown.

**Expected result:**
1. Dropdown arrow should have color contrast ratio of at least 3:1 as it's a non textual informative icon.
2. Also, this drodown should be clearly visible in high contrast mode.

**Actual result:**
1. Color contrast ratio of the dropdown icon is 1.7:1
2. Dropdown arrow is not visible in high contrast mode, which makes it looks like a edit box. 

**Note:** Same issue is observed for Select Clarity Component.

**Reference screenshot**
![dropdown - screenshot]( https://imgur.com/a/OWuLNdy)
![high contrast - screenshot](https://imgur.com/a/5ne8FYg)
